test/integration/client/client_test.go

```diff
@@ -1049,12 +1047,16 @@ func TestExtractModifyApply(t *testing.T) {
 			}
 
 			if tc.modifyStatusFunc != nil {
+				extractedDeployment, err := appsv1ac.ExtractDeploymentStatus(actual, fieldMgr)
+				if err != nil {
+					t.Fatalf("Failed to extract: %v", err)
+				}
 				tc.modifyStatusFunc(extractedDeployment)
 				result, err := deploymentClient.ApplyStatus(context.TODO(), extractedDeployment, metav1.ApplyOptions{FieldManager: fieldMgr})
 				if err != nil {
 					t.Fatalf("Failed to apply extracted apply configuration to status: %v", err)
 				}
-				extractedResult, err := appsv1ac.ExtractDeployment(result, fieldMgr)
+				extractedResult, err := appsv1ac.ExtractDeploymentStatus(result, fieldMgr)
```

Comment:
You forgot to uncomment the test in this file, probably a mix-up during rebase?

Reply:
Good catch! Fixed. It's passing locally.

Reply:
Awesome, thanks!